index.html

```diff
@@ -11240,6 +11278,9 @@ <h2>A project for first-time contributions</h2>
                     </div>
                 </div>
                 <!-- ________ Contributor's card END ________  -->
+
```

Comment:
Hi. Thank you for your PR. You are just one step away before we merge this into the _master_ branch.

It seems like there are a bunch of extra lines that are included in the PR as outlined here. The only change in this PR would be to add your card at the desired location, and no other changes. Could you please remove these extra lines from the PR? Once done, we can get it merged. Thanks.

Reply:
Oh I'm so sorry for the inconvenience , I've made the corrections. Thank you.